app/components/time-ago.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
```

Comment:
could be a helper, i think there's already a `display-datetime` which could be copied/adapted

Reply:
Yeah, i felt the same way. 
The logic was copied from Admin app